extension/js/background_page/migrations.ts

```diff
@@ -40,6 +47,69 @@ export const migrateGlobal = async () => {
   }
 };
 
+const processSmimeKey = (pubkey: Pubkey, tx: IDBTransaction, data: PubkeyMigrationData, next: () => void) => {
+  if (KeyUtil.getKeyType(pubkey.armoredKey) !== 'x509') {
+    next();
+    return;
+  }
```

Comment:
This is interesting - could we instead filter when pulling pubkeys from storage? Eg `SELECT FROM pubkey WHERE type = 'x509'`. Or we don't store this info?

Reply:
We don't store this info yet, only in ids (fingerprints) -- postfixed with "-X509", and in `longids` -- prefixed with "X509-"